lib/phoenix_live_view/router.ex

```diff
@@ -49,11 +49,15 @@ defmodule Phoenix.LiveView.Router do
     quote do
       Phoenix.Router.get(
         unquote(path),
-        Phoenix.LiveView.Controller,
+        Phoenix.LiveView.Plug,
         Phoenix.Router.scoped_alias(__MODULE__, unquote(live_view)),
         private: %{
-          phoenix_live_view: unquote(opts),
-          phoenix_live_view_default_layout: Phoenix.LiveView.Router.__layout_from_router_module__(__MODULE__)
+          phoenix_live_view:
+            Keyword.put(
+              unquote(opts),
+              :default_layout,
```

Comment:
Oh, I just had another idea. What if we call it `:layout` and change this for `Keyword.put_new`? this means folks can set the layout as argument to the `live` call as well (we can document it as an option).

Reply:
I'll make that change 👍.